cv_next/app/actions/users/getUser.ts

```diff
@@ -0,0 +1,25 @@
+"use server";
+
+import { getUserById } from "@/server/api/users";
+import { Err } from "@/lib/utils";
+import { logErrorWithTrace } from "@/server/base/logger";
+
+/**
+ * Retrieves user data by user ID.
+ *
+ * @param {string} userId - The ID of the user to retrieve
+ * @return {Promise<Result<UserModel, string>>} A promise that resolves to a Result containing the user data or an error message
+ */
+export const getUserModel = async (
```

Comment:
Please be consistent with naming - we have `getUserModel`, `fetchCvsForFeed`, `getUserName`.. They all have the same CRUD functionality - a get request. So, preferably, they'd all be named `getX`.

Reply:
In #72 these turned to API calls with only fetchCvsForFeed getting this prefix because it's handled in a different file than the others